hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfoMetrics.java

```diff
@@ -33,6 +35,7 @@ public class VolumeInfoMetrics {
 
   private String metricsSourceName = VolumeInfoMetrics.class.getSimpleName();
   private final HddsVolume volume;
+  private final AtomicLong dbCompactTimesNanoSeconds = new AtomicLong();
```

Comment:
This accumulated ns value cannot tell how many time it's updated.
Please use the MutableRate type as metrics type,  refer to ContainerCacheMetrics.java.

Reply:
> This accumulated ns value cannot tell how many time it's updated. Please use the MutableRate type as metrics type, refer to ContainerCacheMetrics.java.

This is a great suggestion.